### PR TITLE
PHP config and TS

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

File: bad.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function validRange (range: any) {
```
Comment (markstory) [inline bad.ts]: Shadowed name: 'range'
Spaces before function parens are disallowed

File: bad.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  if (range.min < -3) {
+    return false;
+  }
+  return range.min <= range.middle && range.middle <= range.max;
+}
+
+// Unsorted keys
+const range = {
+  min: 5,
+  middle: 10,
```
Comment (markstory) [inline bad.ts]: The key 'middle' is not sorted alphabetically

File: bad.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  max: 20
```
Comment (markstory) [inline bad.ts]: Missing trailing comma

File: bad.php [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Silly;
+
+class Silly_Thing {
```
Comment (markstory) [inline bad.php]: Class name "Silly_Thing" is not in camel caps format
Opening brace of a class must be on the line after the definition

File: bad.php [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    public function badName() {
```
Comment (markstory) [inline bad.php]: Opening brace should be on a new line

File: bad.php [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+        return 'this is the song that goes on and on this is the song that goes on and on this is the song that goes on and on this is the song that goes on and on';
```
Comment (markstory) [inline bad.php]: Line exceeds 120 characters; contains 165 characters

File: bad.ts (hunk 1 above)
Comment (stickler-ci-dev) [inline bad.ts]: Shadowed name: 'range'
Spaces before function parens are disallowed

File: bad.ts (hunk 2 above)
Comment (stickler-ci-dev) [inline bad.ts]: The key 'middle' is not sorted alphabetically

File: bad.ts (hunk 3 above)
Comment (stickler-ci-dev) [inline bad.ts]: Missing trailing comma

File: bad.php (hunk 4 above)
Comment (stickler-ci-dev) [inline bad.php]: Class name "Silly_Thing" is not in camel caps format
Opening brace of a class must be on the line after the definition

File: bad.php (hunk 5 above)
Comment (stickler-ci-dev) [inline bad.php]: Opening brace should be on a new line

File: bad.php (hunk 6 above)
Comment (stickler-ci-dev) [inline bad.php]: Line exceeds 120 characters; contains 165 characters

**Pull request closed without merging**